packages/ingest-mongodb-public/src/sources/snooty.ts

```diff
@@ -0,0 +1,258 @@
+  },
+  {
+    type: "snooty",
+    name: "mongocli",
+    tags: ["docs", "cli", "mongocli"],
+    productName: "MongoDB CLI",
+  },
+  {
+    type: "snooty",
+    name: "mongodb-shell",
+    tags: ["docs", "cli", "mongodb-shell"],
+    productName: "MongoDB Shell",
+  },
+  {
+    type: "snooty",
+    name: "mongodb-vscode",
+    tags: ["docs", "mongodb-vscode", "vscode", "gui"],
+    productName: "MongoDB for VS Code",
+  },
+  {
+    type: "snooty",
+    name: "mongoid",
+    tags: ["docs", "driver", "mongoid", "ruby"],
+    productName: "Mongoid ODM",
+  },
+  {
+    type: "snooty",
+    name: "node",
+    tags: ["docs", "driver", "node", "javascript"],
+    productName: "Node.js Driver",
+  },
+  {
+    type: "snooty",
+    name: "php-library",
+    tags: ["docs", "driver", "php", "php-library"],
+    productName: "PHP Library",
+  },
+  {
+    type: "snooty",
+    name: "laravel",
+    tags: ["docs", "driver", "php", "laravel"],
+    productName: "Laravel MongoDB",
+  },
+  {
+    type: "snooty",
+    name: "realm",
+    tags: ["docs", "realm", "mobile", "sdk"],
+    productName: "Realm SDKs",
+  },
+  {
+    type: "snooty",
+    name: "docs-relational-migrator",
+    tags: ["docs", "relational-migrator"],
+    productName: "MongoDB Relational Migrator",
+  },
+  {
+    type: "snooty",
+    name: "ruby-driver",
+    tags: ["docs", "driver", "ruby"],
+    productName: "Ruby Driver",
+  },
+  {
+    type: "snooty",
+    name: "spark-connector",
+    tags: ["docs", "spark-connector", "spark", "apache-spark"],
+    productName: "MongoDB Spark Connector",
+  },
+  {
+    type: "snooty",
+    name: "guides",
+    tags: ["docs", "guides", "tutorial"],
+  },
+  {
+    type: "snooty",
+    name: "visual-studio-extension",
+    tags: ["docs", "visual-studio-extension", "visual-studio", "gui"],
+    productName: "MongoDB Visual Studio Extension",
+  },
+  {
+    type: "snooty",
+    name: "rust",
+    tags: ["docs", "driver", "rust"],
+    productName: "Rust Driver",
+  },
+  {
+    type: "snooty",
+    name: "cpp-driver",
+    tags: ["docs", "driver", "cpp", "cxx", "c++"],
+    productName: "C++ Driver",
+  },
+  {
+    type: "snooty",
+    name: "c",
+    tags: ["docs", "driver", "c", "libmongoc"],
+    productName: "C Driver",
+  },
+  {
+    type: "snooty",
+    name: "scala",
+    tags: ["docs", "driver", "scala"],
+    productName: "Scala Driver",
+  },
+  {
+    type: "snooty",
+    name: "java-rs",
+    tags: ["docs", "driver", "java", "java-reactive-streams"],
+    productName: "Java Reactive Streams Driver",
+  },
+  {
+    type: "snooty",
+    name: "pymongo",
+    tags: ["docs", "driver", "python", "pymongo"],
+    productName: "PyMongo",
+  },
+  {
+    type: "snooty",
+    name: "pymongo-arrow",
+    tags: ["docs", "driver", "python", "pymongo-arrow"],
+    productName: "PyMongo Arrow",
+  },
+];
+
+export const snootyDataApiBaseUrl = "https://snooty-data-api.mongodb.com/prod/";
+
+export const snootySourceConstructor = (
```

Comment:
```suggestion
export const makeSnootyDataSources = (
```

This matches our general make pattern, e.g. on `makeDevCenterDataSource`.

Left suggestions throughout for this update.